File: Cricket Match.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int n,m;
        scanf("%d%d",&n,&m);
        if(n<=m*36)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: Cricket Match.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-->0)
        {
            int n=sc.nextInt();
            int m=sc.nextInt();
            if(n<=m*36)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}